Moderation-Policy.md

```diff
@@ -1,154 +1,288 @@
+submitting the request should be omitted from the issue unless permission to
+include the identity is provided by the reporter.
 
-Requests should contain as much information and context as possible, including the URL and a screenshot of the Post in question. Screenshots may be modified to obscure obscene or offensive content.
+Requests should contain as much information and context as possible, including
+the URL and a screenshot of the Post in question. Screenshots may be modified
+to obscure obscene or offensive content.
 
-External public venues or social media services such as Twitter should never be used to request Moderation.
+External public venues or social media services such as Twitter should never be
+used to request Moderation.
 
-Collaborators should never discuss the specific details of a Moderation request in any public forum or any social media service outside of the Node.js GitHub Organization.
+Collaborators must never discuss the specific details of a Moderation request
+in any public forum or any social media service outside of the Node.js GitHub
+Organization.
 
-Note that quoting the original content of a Post within a Moderation request or nodejs/moderation repository issue is not considered a violation of the [Code of Conduct][]. However, discretion is advised when including such quotes in requests posted to public repositories.
+Note that quoting the original content of a Post within a Moderation request or
+nodejs/moderation repository issue is not considered a violation of the
+[Code of Conduct][]. However, discretion is advised when including such quotes
+in requests posted to public repositories.
 
-Requests for Moderation that do not appear to have been submitted in good faith with intent to address a legitimate [Code of Conduct][] violation, as determined by the TSC, may be ignored.
+Requests for Moderation that do not appear to have been submitted in good faith
+with intent to address a legitimate [Code of Conduct][] violation may be
+ignored.
 
-### Consideration of Intent
+## Consideration of Intent
 
-Before Moderating a Post, Collaborators should carefully consider the possible intent of the author. It may be that the author has simply made an error or is not yet familiar with the [Code of Conduct][]; or it may be that cultural differences exist, or that the author is unaware that certain content is considered inappropriate. In such cases, the author should be given an opportunity to correct any error that may have been made.
+Before Moderating a Post, Collaborators should carefully consider the possible
+intent of the author. It may be that the author has simply made an error or is
+not yet familiar with the [Code of Conduct][]; or it may be that cultural
+differences exist, or that the author is unaware that certain content is
+considered inappropriate. In such cases, the author should be given an
+opportunity to correct any error that may have been made.
 
```

Comment:
perhaps a note about privately contacting an individual being a valid option if you think the intent may have been good, but the result bad.

This is a good learning opportunity, and would be good to nudge people towards a kinder approach